django_mongodb_engine/base.py

```diff
@@ -69,6 +74,8 @@ def sql_flush(self, style, tables, sequence_list, allow_cascade=False):
             options = collection.options()
 
             if not options.get('capped', False):
+
+                # TODO:Not backwards compatible
```

Comment:
A comment i'd forgotten to remove after making this backwards compatible. Was using the new pymongo 3 deleteMany